trace-utils/src/trace_utils.rs

```diff
@@ -17,6 +17,8 @@ const TOP_LEVEL_KEY: &str = "_top_level";
 /// Span metric the tracer sets to denote a top level span
 const TRACER_TOP_LEVEL_KEY: &str = "_dd.top_level";
 
+const MAX_PAYLOAD_SIZE: usize = 52428800;
```

Comment:
Could we use `_` or multiplication in long numbers, like `52_428_800` or `50 * 1024 * 1024`?

Reply:
Yes, absolutely.